src/config/github-api.ts

```diff
@@ -1,12 +1,11 @@
-import { GitHubAPI } from "probot";
+import { GitHubAPI as ProbotGitHubAPI } from "probot";
 import { getLogger } from "./logger";
-import { enhanceOctokit } from "./enhance-octokit";
 import { Options } from "probot/lib/github";
 
-export const GithubAPI = (options: Partial<Options> = {}): GitHubAPI => {
+export const GithubAPI = (options: Partial<Options> = {}): ProbotGitHubAPI => {
```

Comment:
Is anything still using this reference?  I thought we were only using our github clients from now on?

Reply:
in the webhooks theres reference to it, shouldn't be in use but if i remove this the thread it pulls makes the PR way bigger so i split it in to another PR.